lib/cdo/slack.rb

```diff
@@ -148,6 +148,27 @@ def self.message(text, params={})
     end
   end
 
+  # For more information see
+  # https://github.com/ErikKalkoken/slackApiDoc/blob/master/chat.command.md.
+  # @param channel_id [String] Id of the Slack channel to post the command to.
+  # You can find the channel id at the end of the url for a given channel using
+  # Slack in the browser, e.g https://codedotorg.slack.com/messages/<channel_id>
+  # @param command [String] Command to execute, excluding the /.
+  # @param message [String] Optional text passed to the command.
+  # @return [Boolean] Whether the command was posted to Slack successfully.
```

Comment:
It's worth noting, these are officially undocumented APIs. See https://github.com/ErikKalkoken/slackApiDoc. They "may change at any time, you should not rely on these behaviors."

Reply:
I'd add that it's definitely worth noting that (and even probably doing so explicitly in the comment), but that it's probably still fine for us to use it for this purpose

Reply:
Agreed. To be clear, I'm suggesting we note that and wrap it in a more general API (see below).